components/_form-choice.scss

```diff
@@ -23,18 +23,20 @@
 
 .form-choice__label {
     position: relative;
-    display: inline-flex;
-    align-items: center;
+    display: inline-block;
     min-height: 1.5rem;
     margin-bottom: 0;
+    padding-left: 1.5rem;
     font-family: $form-choice-font-family;
     cursor: pointer;
 
     &:before {
         content: '';
+        position: absolute;
+        top: 3px;
```

Comment:
Kun je dit niet met em doen? Want wanneer het lettertype groter word is dit niet het midden.